### ISSUE TITLE
Remove usage of deprecated std::slice::bytes::copy_memory

### ISSUE THREAD

**Issue by alex** [servo/servo#8929]
```
/Users/alex_gaynor/projects/servo/components/canvas/webgl_paint_task.rs:16:5: 16:35 warning: use of deprecated item: unidiomatic functions not pulling their weight, #[warn(deprecated)] on by default
/Users/alex_gaynor/projects/servo/components/canvas/webgl_paint_task.rs:16 use std::slice::bytes::copy_memory;
                                                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/alex_gaynor/projects/servo/components/canvas/webgl_paint_task.rs:368:13: 368:24 warning: use of deprecated item: unidiomatic functions not pulling their weight, #[warn(deprecated)] on by default
/Users/alex_gaynor/projects/servo/components/canvas/webgl_paint_task.rs:368             copy_memory(&src_slice[..stride], &mut pixels[dst_start .. dst_start + stride]);
                                                                                        ^~~~~~~~~~~

/Users/alex_gaynor/projects/servo/components/compositing/compositor.rs:47:5: 47:35 warning: use of deprecated item: unidiomatic functions not pulling their weight, #[warn(deprecated)] on by default
/Users/alex_gaynor/projects/servo/components/compositing/compositor.rs:47 use std::slice::bytes::copy_memory;
                                                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/alex_gaynor/projects/servo/components/compositing/compositor.rs:1916:13: 1916:24 warning: use of deprecated item: unidiomatic functions not pulling their weight, #[warn(deprecated)] on by default
/Users/alex_gaynor/projects/servo/components/compositing/compositor.rs:1916             copy_memory(&src_slice[..stride],
                                                                                        ^~~~~~~~~~~
```

**Comment by KiChjang** [comment]
I propse using the `write` function as implemented by the `Write` trait on `Vec<u8>`, `&[u8]` and `&,it [u8]`.

**Comment by jinwooahn** [comment]
Can I be assigned? My patch is almost ready. :)

**Comment by Wafflespeanut** [comment]
@jinwooahn Sorry, it's being fixed now (probably because it's urgent) - you can try the other E-easy issues :)

**Comment by KiChjang** [comment]
Fixed by #8937.
